Index: nslu2_protocol.h
===================================================================
--- nslu2_protocol.h	(revision 39)
+++ nslu2_protocol.h	(working copy)
@@ -183,7 +183,7 @@
 		MaxPacketLength = 1540-14,   /* at least 600 from the RedBoot code */
 		BaseAddress = 0x60000,       /* skip RedBoot and SysConf */
 		UpgradeProtocol = 0x8888,    /* defined in the RedBoot code */
-		Ln2FlashSize = 23,           /* 8MByte Flash memory */
+		Ln2FlashSize = 24,           /* 16MByte Flash memory */
 		FlashSize = (1<<Ln2FlashSize),
 		/* The MaxPendingPackets figure is the number of packets which will
 		 * be transmitted without receiving a response, this value should be
